tests/manage/cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -0,0 +1,23 @@
+import logging
+
+from tests import helpers
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs import machine as machine_utils
+from ocs_ci.ocs import constants, ocp
+
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
haven't seen we're doing a teardown to decrease osd count

Reply:
Tried manually, did not work out, will dig more

Reply:
in that case, could you please add the functionality for teardown and we'll block this TC with a bugzilla decorator in case of a bug or think of a workaround in case it's not